extension/test/background_test.ts

```diff
@@ -65,26 +87,76 @@ describe('background.ts', function () {
         { config: rcxMain.config }
       );
     });
+
+    it('should throw typeError if tab is undefined', async function () {
+      // }).to.throw(TypeError, 'sender.tab is always defined here.');
```

Comment:
it's not removed yet. :)

I recommend only marking conversations as resolved after you've pushed since otherwise reviewers can't easily confirm the solution.